docs/development/setup-staging-release.md

```diff
@@ -49,7 +49,7 @@ Optional: reset the staging balrog db which is important for staging release bui
 
 - release runner will consume [project branch configs](https://dxr.mozilla.org/build-central/rev/92614acc90330edf360d97d8575b7e917ddc43b2/buildbot-configs/mozilla/project_branches.py#114)
 - mozharness [configs](https://dxr.mozilla.org/mozilla-central/source/testing/mozharness/configs/releases/) - all configs with `dev` prefix
-- in order to avoid pushing patcher configs changes/tags to *real* [tools](http://hg.mozilla.org/build/tools/) repo, we have changed the buildbot and in-tree configs to point to the <a href="https://hg.mozilla.org/users/stage-ffxbld/tools">staging tools repo</a>) 
+- in order to avoid pushing patcher configs changes/tags to *real* [tools](http://hg.mozilla.org/build/tools/) repo, we have changed the buildbot and in-tree configs to point to the <a href="https://hg.mozilla.org/users/stage-ffxbld/tools">staging tools repo</a>)
```

Comment:
Do we actually push the configs anywhere now, or do we just use task artifacts?

Reply:
Nope, but I didn't want to specifically start cleaning up in this this file within this PR. But I agree, we can most lilkely wipe this `Staging configs` section modulo the mozharness configs line.

Reply:
Hm, come to think about it - a more general cleanup is needed now that esr52 is EOL, Ship-it v2 is ramping up, tcmigration is done, etc. I filed https://github.com/mozilla-releng/releasewarrior-2.0/issues/182 to track this (more general) cleanup.